test-integration/graql/query/QueryPlannerIT.java

```diff
@@ -398,6 +399,11 @@ public void sameLabelFragmentShouldNotBeAddedTwice() {
                 .filter(fragment -> fragment instanceof OutIsaFragment || fragment instanceof InIsaFragment).count());
     }
 
+    /**
+     * We no longer rely on shard count to order query plan starting points, using instance counts directly
+     * Whether we want to delete this test is not yet clear, or just keep a similar one
+     */
+    @Ignore
```

Comment:
If it is not relevant anymore we should probably remove it. Alternatively replace it with an appropriate equivalent.